### ISSUE TITLE
[Integrity Checker] : Improving error message when fixing File Asset conflict

### ISSUE THREAD

**Issue by jcastro-dotcms** [dotCMS/core#20232]
Found when working on Support ticket: [103967](https://dotcms.zendesk.com/agent/tickets/103967)
Reproducible in **ALL** versions of dotCMS.

When fixing File Asset conflicts, there might be situations in which the incoming Inode that must be used to fix the conflict already exists. In this case, we need to log the appropriate error message, and cancel the fix operation. This type of conflict must be strictly inspected by a dotCMS developer before applying any fix.

**To Reproduce**
Steps to reproduce are not clear. All we know is that, at some point, the incoming Inode already existed in the server where the conflict is being resolved. The error in the `dotcms.log` file is the following:
```
[12/04/21 16:29:50:056 GMT] ERROR integritycheckers.ContentFileAssetIntegrityChecker: Error while copying the assets to the new folder.  Info: old inode=[56070968-d4c1-4969-b9b1-fbfbd78ec407] - new inode=[548cfe92-735f-4ad2-98f0-b444142cbcec]
[12/04/21 16:29:50:057 GMT] ERROR rest.IntegrityResource: Error fixing fileAssets conflicts for End Point server: [23ca0a3a-14d0-4b6c-8d23-97213a7e0a17]
com.dotmarketing.business.DotStateException: Error while copying the assets to the new folder.  Info: old inode=[56070968-d4c1-4969-b9b1-fbfbd78ec407] - new inode=[548cfe92-735f-4ad2-98f0-b444142cbcec]
	at com.dotcms.integritycheckers.ContentFileAssetIntegrityChecker.moveInodeFolder(ContentFileAssetIntegrityChecker.java:571) ~[dotcms_5.3.8.4_999999.jar:?]
	at com.dotcms.integritycheckers.ContentFileAssetIntegrityChecker.generateNewContentlet(ContentFileAssetIntegrityChecker.java:474) ~[dotcms_5.3.8.4_999999.jar:?]
	at com.dotcms.integritycheckers.ContentFileAssetIntegrityChecker.fixContentletConflicts(ContentFileAssetIntegrityChecker.java:404) ~[dotcms_5.3.8.4_999999.jar:?]
	at com.dotcms.integritycheckers.ContentFileAssetIntegrityChecker.executeFix(ContentFileAssetIntegrityChecker.java:108) ~[dotcms_5.3.8.4_999999.jar:?]
	at com.dotcms.integritycheckers.IntegrityUtil.fixConflicts(IntegrityUtil.java:580) ~[dotcms_5.3.8.4_999999.jar:?]
	at com.dotcms.rest.IntegrityResource.fixConflicts(IntegrityResource.java:1118) ~[dotcms_5.3.8.4_999999.jar:?]
...
...
```

**Expected behavior**
If a conflict cannot be fixed, the Integrity Checker should at least displaying a meaningful error describing the situation.

**Additional context**
Effective error reporting is crucial for the Enterprise Support Team.

**Comment by fabrizzio-dotCMS** [comment]
Found a small error formatting an error message that gets corrected with this PR
https://github.com/dotCMS/core/pull/20289

**Comment by fabrizzio-dotCMS** [comment]
I forced the error through my debugger and it looks good to me.

**Comment by bryanboza** [comment]
Fixed, tested on release-21.05 // Postges // FF. This on both servers and I was unable to reproduce the error
